### PR TITLE
Standardize disk requirements, time requirements, and package counts

### PR DESCRIPTION
**Pull request by electronwill**
Standardize disk requirements, time, requirements, and package counts;
they're still not exact but at least they're accurate and consistent.

### CODE REVIEW

**Comment by electronwill** [comment]
@asmeurer @kerrywatson1 ready to merge

**Comment by kerrywatson1** [comment]
Love it, please merge.

File: docs/source/download.rst [hunk 1]
```diff
@@ -17,16 +17,16 @@ Should I download Anaconda or Miniconda?
...
+* Don’t want to install each of the packages you want to use individually. 
 
 Anaconda download: http://continuum.io/downloads
 
 **Choose Miniconda if you:**
 
-* Do not mind installing about 100 programs each individually
-* Do not have time or disk space to install 100 programs at once, and/or
+* Do not mind installing each of the packages you want to use individually
+* Do not have time or disk space to install over 100 programs at once, and/or
```
Comment (groutr) [inline docs/source/download.rst]: should it read "100 packages"?

Comment (kerrywatson1) [inline docs/source/download.rst]: yes.

Comment (electronwill) [inline docs/source/download.rst]: ok, changing that now

**Pull request merged**